app/components/ui/molecules/PeoplePicker.test.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ThemeProvider } from 'styled-components'
+import theme from '@elifesciences/elife-theme'
+import PeoplePicker from './PeoplePicker'
+
+const people = [
+  { id: 1, name: 'Annie' },
+  { id: 2, name: 'Bobby' },
+  { id: 3, name: 'Chastity' },
+  { id: 4, name: 'Dave' },
+]
+
+const makeWrapper = props =>
+  mount(
+    <ThemeProvider theme={theme}>
+      <PeoplePicker
+        // eslint-disable-next-line react/no-children-prop
+        children={innerProps => <PeoplePicker.Body {...innerProps} />}
```

Comment:
Do we need this? Rather than disabling the eslint rule, we could just explicitly nest the `PeoplePicker.Body` inside the `PeoplePicker` couldn't we?

Reply:
We do need to do it this way in order to override the `children` prop in some tests. I'll add a comment.